WordPress/Classes/Utility/Universal Links/Routes+Reader.swift

```diff
@@ -106,6 +109,10 @@ extension ReaderRoute: NavigationAction {
             if let (blogID, postID) = blogAndPostID(from: values) {
                 coordinator.showPost(with: postID, for: blogID, isFeed: false)
             }
+        case .wpcomPost:
+            if let urlString = values["matched-route-url"], let url = URL(string: urlString) {
```

Comment:
You can use `MatchedRouteURLComponentKey.url` here, instead of the literal value.